src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java

```diff
@@ -184,4 +187,19 @@ private static StandardCredentials resolveCredentials(@CheckForNull String crede
         }
         return c;
     }
+
+    private static class WithContextClassLoader implements AutoCloseable {
```

Comment:
This is available in core. (Not sure about your current baseline offhand, but at least leave a TODO.)

Reply:
https://github.com/jenkinsci/kubernetes-plugin/pull/1289/files#diff-fe2f8caf9a928226f5d0a4ba20ff6a57846f75fcec2fcc925962cf87fd8f4040R147